### ISSUE TITLE
Update ROS-ESM package links

### ISSUE THREAD

**Issue by mtruj013** [canonical/ubuntu.com#12154]
When a CVE has a package named `ros-kinetic-ros-comm` or `ros-melodic-ros-comm`, the template should not have the Launchpad, Ubuntu, or Debian links. Instead it should link to https://ubuntu.com/robotics/ros-esm  , link text should be 
`ROS ESM`
<img width="845" alt="Screenshot 2022-10-11 at 17 55 57" src="https://user-images.githubusercontent.com/17607612/195141205-061c8a3e-88ad-4055-b958-c21a6fee7986.png">

You can QA against this CVE: https://ubuntu.com/security/CVE-2021-37146
Json: https://ubuntu.com/security/cves/CVE-2021-37146.json

**Comment by florcabral** [comment]
Hello, as the person who requested this issue, I wanted to make a clarification on the need:
The packages that should have their links replaced in this way are all packages included in these 2 lists:
* Xenial (Kinetic) packages: https://git.launchpad.net/ubuntu-cve-tracker/tree/ros-esm-xenial-kinetic-supported.txt
* Bionic (Melodic) packages: https://git.launchpad.net/ubuntu-cve-tracker/tree/ros-esm-bionic-melodic-supported.txt

That is, those supported for the Xenial and Bionic releases, where the naming structure is `ros-<distro>-<package-name>`.
In other words, this ticket applies to `ros-comm` (in its `ros-kinetic-ros-comm` and `ros-melodic-ros-comm` versions), as well as to every other package in these lists. Please let me know if I can answer any questions. Thank you.
